gitgud/levels/reworking/__init__.py

```diff
@@ -0,0 +1,9 @@
+from collections import OrderedDict
+
+import pkg_resources
+
+from gitgud.levels.util import BasicChallenge
+
+all_challenges = OrderedDict()
+all_challenges['cherrypick'] = BasicChallenge('cherrypick', pkg_resources.resource_filename(__name__, '_cherrypick/'))
+all_challenges['interactiverebase'] = BasicChallenge('interactiverebase', pkg_resources.resource_filename(__name__, '_interactiverebase/'))
```

Comment:
Can you rename this to 'rebase-i'? "interactiverebase" is a bit tedious to type in. Change the folder location as well

Reply:
irebase might be better. We're starting to use dashes for loading the level as in #105